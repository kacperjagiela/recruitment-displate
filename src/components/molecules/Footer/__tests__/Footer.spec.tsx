import { render } from '@testing-library/react';

import { Footer } from '..';

describe('<Footer />', () => {
    it('renders correctly', () => {
        const { container } = render(<Footer />);

        expect(container).toMatchSnapshot();
    });
});
