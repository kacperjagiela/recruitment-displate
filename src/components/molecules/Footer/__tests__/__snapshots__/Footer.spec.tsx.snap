// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<div>
  <h5
    class="chakra-text css-6uhn4g"
  >
    Developed by
     
    <a
      class="chakra-link css-1jk86wo"
      href="https://github.com/kacperjagiela"
    >
      Kacper Jagie≈Ça
    </a>
     
    for
     
    <a
      class="chakra-link css-1jk86wo"
      href="https://displate.com"
    >
      Displate
    </a>
  </h5>
</div>
`;
